### DISCUSSION TITLE
Segmentation faults.

### DISCUSSION THREAD

**Discussion by Xorrongo** [ggml-org/llama.cpp#471] [General]
Using: M1 Macbook Pro, 16GB ram.  Latest code.  Using the 4bit models (7 or 13B) I reliably get segmentation faults after a few paragraphs of use. It doesn't seem to be an out of memory error ( there is plenty of swap available.) Any ideas?

**Comment by swittk** [comment]
Can confirm this occurs to me too; M1 Max 32GB Ram, Master branch.
